DOCS-10787 Amazon Redshift 1.0

valkyrie69 opened the pull request.

Commit:
DOCS-10787 Amazon Redshift 1.0

Commit:
Added more info

Commit:
add guides for Anypoint Studio and Maven

Commit:
add example

Commit:
Edits

Commit:
Added missing descriptions

Commit:
Edits

dhirsch1 commented on amazon-redshift/1.0/modules/ROOT/pages/amazon-redshift-connector-example.adoc:
```diff
@@ -0,0 +1,231 @@
+
+[NOTES]
+---
+* You must have S3 credentials to create and delete buckets and to create an object.
```
To fix later: S3 > Amazon S3 here and where needed
Same with Redshift > Amazon Redshift

dhirsch1 commented on amazon-redshift/1.0/modules/ROOT/pages/amazon-redshift-connector-example.adoc:
```diff
@@ -0,0 +1,231 @@
+* Alternatively, it is possible to use an IAM Role instead of the Access and Secret key pair. 
++
+To do this, you must ensure the role has the correct policies attached to it. It is best to use the Amazon S3 Read only policy for this purpose.
+** To create an IAM role to allow your Redshift cluster to communicate with the Amazon S3 service on your behalf, follow the steps in this https://docs.aws.amazon.com/redshift/latest/mgmt/authorizing-redshift-service.html[Amazon Redshift tutorial].
```
This bullet didn't format correctly, maybe because of a missing + above it.

dhirsch1 commented on amazon-redshift/1.0/modules/ROOT/pages/amazon-redshift-connector-reference.adoc:
```diff
@@ -0,0 +1,1087 @@
+= Amazon Redshift Connector 1.0 Reference
+
+Anypoint Connector for Amazon Redshift (Amazon Redshift Connector) is a bulk-load connector that enables you to connect to Amazon Redshift database to perform inserts, deletes, updates, and to execute SQL queries. 
```
"connect to the Amazon Redshift database"

dhirsch1 commented on amazon-redshift/1.0/modules/ROOT/pages/amazon-redshift-connector-studio.adoc:
```diff
@@ -0,0 +1,135 @@
+
+* Basic authentication, which uses *user* and *password* field values to connect to the database
+* IAM authentication, which uses *access key* and *secret key* field values to retrieve temporary password for the specified user in order to connect to the database
+** Optionally, you can specify *ARN* field value in the *Role* field if the connector should first assume an IAM role for a set of temporary security IAM credentials and use those to retrieve the password.
```
Formatting is funky - this should be a list continue and not a second-level bullet. 

"you can specify *ARN* field value" > "you can specify the *ARN* field value"

dhirsch1 commented on amazon-redshift/1.0/modules/ROOT/pages/amazon-redshift-connector-studio.adoc:
```diff
@@ -0,0 +1,135 @@
+A source initiates a flow when a specified condition is met.
+You can configure the following source to use with Amazon Redshift instance:
+
+* *On Table Row*
```
For later, this way of listing sources does not match the template.

dhirsch1 commented on amazon-redshift/1.0/modules/ROOT/pages/amazon-redshift-connector-studio.adoc:
```diff
@@ -0,0 +1,135 @@
+. Select the created connector configuration or click the plus sign (*+*) next to the *Connector _configuration_* field to configure a new global element.
+Follow the steps in <<configure-global-element,Configure a global element for the connector>>.
+. On the *General* tab, complete the following fields:
+
```
Missing +

Commit:
Peer review edits

Commit:
remove encoding parameter

Commit:
- update redshift-configuration.png image
- fix runtime version in antora.yml
- add missing information regarding configuration of JDBC driver to docs

Commit:
- fix titles
- fix broken link

Commit:
- fix outdated text

dhirsch1 committed:
Edited new dev changes

dhirsch1 committed:
Minor fix

dhirsch1 committed:
Bolded Required libraries

Commit:
add missing step to tutorials

Commit:
Merge remote-tracking branch 'origin/DOCS-10787-amazon-redshift-1.0' into DOCS-10787-amazon-redshift-1.0

# Conflicts:
#	amazon-redshift/1.0/modules/ROOT/pages/amazon-redshift-connector-example.adoc
#	amazon-redshift/1.0/modules/ROOT/pages/amazon-redshift-connector-studio.adoc

Commit:
reword configure JDBC driver tutorial step

dhirsch1 committed:
Edited a dev text addition

dhirsch1 committed:
Formatting fixes

Commit:
CONN-8018 - fix minimum Anypoint Studio version

Commit:
Merge remote-tracking branch 'origin/DOCS-10787-amazon-redshift-1.0' into DOCS-10787-amazon-redshift-1.0

kevintroller commented on amazon-redshift/1.0/modules/ROOT/pages/amazon-redshift-connector-example.adoc:
```diff
@@ -0,0 +1,244 @@
+
+* Java 8 or 11
+* Anypoint Studio 7.5 and later
+* Mule Runtime 4.3.0 and later
```
```suggestion
* Mule runtime engine (Mule) 4.3.0 and later
```

kevintroller commented on amazon-redshift/1.0/modules/ROOT/pages/amazon-redshift-connector-studio.adoc:
```diff
@@ -0,0 +1,158 @@
+Amazon Redshift Connector supports:
+
+* Basic authentication, which uses *user* and *password* field values to connect to the database
+* IAM authentication, which uses *access key* and *secret key* field values to retrieve temporary password for the specified user in order to connect to the database
```
```suggestion
* IAM authentication, which uses *access key* and *secret key* field values to retrieve the temporary password for the specified user in order to connect to the database.
```

kevintroller commented on amazon-redshift/1.0/modules/ROOT/pages/amazon-redshift-connector-studio.adoc:
```diff
@@ -0,0 +1,158 @@
+
+Amazon Redshift Connector supports:
+
+* Basic authentication, which uses *user* and *password* field values to connect to the database
```
```suggestion
* Basic authentication, which uses *user* and *password* field values to connect to the database.
```

dhirsch1 committed:
Update amazon-redshift/1.0/modules/ROOT/pages/amazon-redshift-connector-example.adoc

Co-authored-by: Kevin Troller <47541849+kevintroller@users.noreply.github.com>

dhirsch1 committed:
Minor edit

dhirsch1 committed:
Minor edit

dhirsch1 committed:
Merge branch 'latest' into DOCS-10787-amazon-redshift-1.0
